lib/testing/src/lib/content-services/rest-apis/nodes/node-content-tree.ts

```diff
@@ -0,0 +1,79 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NodeBodyCreate, NODE_TYPE_FILE, NODE_TYPE_FOLDER, NODE_TITLE, NODE_DESCRIPTION } from './node-body-create';
+
+export interface NodeContentTree {
+    name?: string;
+    files?: string[];
+    folders?: (string|NodeContentTree)[];
+    title?: string;
+    description?: string;
+}
+
+export function flattenNodeContentTree(content: NodeContentTree, relativePath: string = '/'): NodeBodyCreate[] {
```

Comment:
not sure if this is the right place to export flattenNodeContentTree

Reply:
do you have another suggestion where should this be placed and exported?